src/main/java/org/blondin/fa/Main.java

```diff
@@ -39,15 +39,16 @@ private Main() {
     public static void main(String[] args) throws IOException {
 
         if (args == null || args.length < ARGS_MIN) {
-            exitWithError("Usage : java -jar files-archiver-x.y.z.jar inputDirectory outputDirectory fileType1,fileType2 [patternForbidden1,patternForbiddenX]");
+            exitWithError(
+                    "Usage : java -jar files-archiver-x.y.z.jar inputDirectory outputDirectory fileType1,fileType2 [patternForbidden1,patternForbiddenX]");
         }
 
         LOG.info("--- Files Archiver ---");
         LOG.info("In directory : " + args[0]);
         LOG.info("Out directory : " + args[1]);
         LOG.info("Files type : " + args[2]);
         if (args.length >= ARGS_MIN) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) NullPointerException might be thrown as 'args' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2259)